plugins/index.js

```diff
@@ -6,10 +6,14 @@ module.exports = (on, config) => {
   } else if (extension === 'keplr') {
     selectedConfig = require('./keplr-plugin');
   } else {
-    throw new Error(
-      `${extension} is not a valid extension name`,
-    );
+    throw new Error(`${extension} is not a valid extension name`);
   }
 
+  if (process.env.SKIP_EXTENSION_SETUP)
+    config.env.SKIP_EXTENSION_SETUP = JSON.parse(
+      process.env.SKIP_EXTENSION_SETUP,
+    );
+  if (process.env.EXTENSION) config.env.EXTENSION = process.env.EXTENSION;
+
```

Comment:
Add curly braces for consistency

Reply:
one liners usually have no braces. 
lets work on a PR for setting standards for linting and fix these kinds of issues in that

Reply:
Sure, let's do it. But in the same file, adjacent if blocks have curly braces. For consistency, it's best to add here as well.